R/utils-json.R

```diff
@@ -72,6 +76,7 @@ getJSON <- function(.pd, evilMode) {
   namedAttrList <- getPDAttributes(.pd)
   class <- attr(.pd, 'class')[1] 
 
+  # why do we care about evilMode here??
```

Comment:
We dont want to return stats when evilMode=T, so i assume this check is a second round of defense (the first round is in the class constructor function i believe)